src/modules/app/appSession.js

```diff
@@ -38,21 +39,33 @@ export class AppSession {
   getBuildNumber() {
     return this.state.buildInfo && this.state.buildInfo.buildInfoBuildNumber;
   }
+
+  /**
+   * Similar to copy on case classes in Scala
+   *
+   * @param props properties that should override properties in appSession state
+   * @returns {AppSession}
+   */
+  copy(props) {
+    return new AppSession({...this.state, ...props});
+  }
 }
 
 const initialState = { accessToken: getAccessToken() };
 
+const orderByMuseumId = (arr, dir = 'desc') => orderBy(arr, ['museumId'], [dir]);
```

Comment:
inline this code. since its never used more than one place, down below to sort groups. The dir parameter will only be desc